### PR TITLE
adds the svg diagram

### PR DESCRIPTION
**Pull request by Amrita42**
### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

[JSC#DOCTEAM-310](https://jira.suse.com/browse/DOCTEAM-310)
[BSC#1188688](https://bugzilla.suse.com/show_bug.cgi?id=1188688)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
- SLE 12
  - [x] SLE 12 SP5
  - [x] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by Amrita42** [comment]
@cwickert ptal, anything else required?

**Review by cwickert — COMMENTED** [review]
tl;dr: I don't think that switching from PNG to SVG is the right approach. Better export the SVG to PNG. While SVG is better for diagrams as it is scalable, some browsers have rendering problems, e.g. they don't get transparency right or don't understand certain SVG instructions. This diagram is not overly complex, so it should work, but generally PNG is the saver option.

Long story, pretty messed up:
1. If you look at sled10_ad_schema.svg, you'll see it was already fixed. According to `git log`  this was done in 959baa946e78e225d91d08894192c09e7b3b9d14 and in the commit message you will see that there was [another bug report for this already](https://bugzilla.suse.com/show_bug.cgi?id=1178719), only for SLE 15 instead of the [SLE 12 bug you are now addressing](https://bugzilla.suse.com/show_bug.cgi?id=1188688).
In that commit however, Carla explicitly added the PNG, but the xml still referenced the SVG.
2. The PNG later got deleted in e3e0038cc670f8b153a698ba35f443da1120f793. I have no idea why, I think it was a mishap.
3. Yet later in 87bbfc22f2fc2ab576ec5b74a37a46ddfc552e06, @janajaeger fixed the SVG and explicitly switched to PNG (again). Jana is out SVG guru, she probably had her reasons to switch. However she did not check if the PNG was really present.

I suggest that you export the SVG to PNG with inkscape and then leave the XML untouched. @janajaeger, what is your take on this?

**Comment by janajaeger** [comment]
It's not black and white with SVG, things and libraries change all the time. For Tahlia's SAP stuff, SVG works perfectly. Test the SVGO optimized SVG. If SVG works both in the browser and PDF, do by all means include SVG. If PNG looks great in both PDF and HTML, take the safe route and reference the file as PNG. Inkscape will then create the PNG automatically upon build, IIRC.

**Review by cwickert — APPROVED** [review]
Okay then, the diagram looks is not too complicated, it okay as SVG, so I assume it's save to use.
